src/main/appengine/queue.yaml

```diff
@@ -64,3 +64,7 @@ queue:
   bucket_size: 10
   retry_parameters:
     min_backoff_seconds: 1
+- name: deadline-extensions-queue
+  mode: push
+  rate: 1/s
+  bucket_size: 5
```

Comment:
Should specify `task_retry_limit` as well (maybe 2 or 3?). For this task worker, I believe it only fails if the paramaters are invalid (or if the fetched feedback session is null). I don't think it's reasonable to retry the task indefinitely since if the parameters are invalid, most likely the task will fail if it is retried.

I think short `min_backoff_seconds` might be reasonable to add as well? In case of any intermittent issues.

Reply:
Yea, this one, I was honestly not sure about this. How do you decide on the values? I think I will just follow what you say about this. For the `min_backoff_seconds`, I think there is already a default value of 0.1 seconds. Is that alright? Or do you want something longer, like maybe 1 second?

Reply:
Hmm maybe 5? I thinking of what could cause a failure, either parameters are incorrect (in which case doesn't matter how many times we retry) or the course/feedback sess "disappears", which might be due to eventual consistency issues? (in which case 5 secs should be more than enough?) 

Honestly, I'm also flying blind here, this is pure guesswork.